Makefile

```diff
@@ -53,6 +53,9 @@ docker-push: docker-push-test-image
 format:
 	go fmt ./...
 
+generate-client:
```

Comment:
Is this target just triggered manually for now?

Reply:
That's correct. If we run this as a dep to other targets, then eventual changes won't be committed.